java/client/src/org/openqa/selenium/remote/DriverCommand.java

```diff
@@ -210,17 +210,14 @@ static CommandPayload SET_ALERT_VALUE(String keysToSend) {
   String SET_ALERT_CREDENTIALS = "setAlertCredentials";
 
   String SET_TIMEOUT = "setTimeout";
-  static CommandPayload SET_IMPLICIT_WAIT_TIMEOUT(long time, TimeUnit unit) {
-    return new CommandPayload(
-        SET_TIMEOUT, ImmutableMap.of("implicit", TimeUnit.MILLISECONDS.convert(time, unit)));
+  static CommandPayload SET_IMPLICIT_WAIT_TIMEOUT(Duration duration) {
```

Comment:
Not sure if I am missing something, but shouldn't this interface use the same (or at least similar) approach as others? 

If I understand this change correctly, someone could be using `CommandPayload SET_IMPLICIT_WAIT_TIMEOUT(long time, TimeUnit unit)`, and if this change was merged and released, and they upgrade, their code would not compile. Is that right or maybe I am missing something?

Reply:
I mean, someone could have implemented their own driver.

Reply:
@diemol you are right, the question is what should be considered as public API and what as internal API, anyway I'll add back old methods

Reply:
Good point, but if the interface is public we cannot discard that some people have implemnted it.